app/README.md

```diff
@@ -49,7 +49,7 @@ To give your Terminal full disk access, go to System Preferences > Security and
 If you see a console error that says the database is malformed, then you need to delete the databases the application reads from (so then it can quickly create them from scratch again).
 
 ```
-cd; rm -ir ./leftonread
```

Comment:
wouldn't the removal of this slash imply that leftonread is not a directory?

Reply:
For some reason, it is actually directory, but doesn't have a slash. Very weird.

Reply:
Weird..